python/src/main/python/drivers/glsl-to-spv-worker.py

```diff
@@ -451,30 +462,24 @@ def main():
     worker_info_file = 'worker_info.json'
     remove(worker_info_file)
 
-    if args.linux:
-        vkrun.dump_info_linux()
+    if is_android:
+        runspv.dump_info_android_legacy(wait_for_screen=not args.force)
     else:
-        vkrun.dump_info_android(wait_for_screen=not args.force)
+        runspv.dump_info_host_legacy()
 
     if not os.path.isfile(worker_info_file):
-        print('Failed to retrieve worker information. '
-              'Make sure the app permission to write to external storage is enabled.')
-        exit(1)
+        raise Exception('Failed to retrieve worker information.  If targeting Android, make sure the app permission '
+                        'to write to external storage is enabled.')
 
-    # noinspection PyUnusedLocal
-    worker_info_json_string = '{}'  # Default value: dummy but valid JSON string
```

Comment:
This sounds like a good idea (and I am guessing I added `noinspection PyUnusedLocal` to ensure there is no warning for this statement), but now that I look more closely, I am guessing that  'worker_info_json_string' will either be overwritten or an exception will be thrown. If we catch exceptions here, then this would make sense and the warning will probably go away.

Reply:
OK I have changed this back and we also now catch and ignore exceptions on the file read.